Editorial review: Document remaining Wasm SIMD instructions (#43800)
mdn/content: files/en-us/webassembly/reference/simd/arithmetic/extmul_high_i16x8_u/index.md

---
title: "extmul_high_i16x8_u: Wasm SIMD arithmetic instruction"
short-title: extmul_high_i16x8_u
slug: WebAssembly/Reference/SIMD/arithmetic/extmul_high_i16x8_u
page-type: webassembly-instruction
browser-compat: webassembly.simd.extmul_high_i16x8_u
sidebar: webassemblysidebar
---

> The **`extmul_high_i16x8_u`** [SIMD arithmetic instruction](/en-US/docs/WebAssembly/Reference/SIMD/arithmetic) takes lanes 4–7 of two unsigned [`v128`](/en-US/docs/WebAssembly/Reference/Types/v128) `i16x8` value interpretations, multiplies the equivalent lanes of each value together, and outputs the products of those operations into an `i32x4` value interpretation.

Reviewer: I found the wording a little awkward. I think this is better - if you agree roll out to the other cases.

```suggestion
The **`extmul_high_i16x8_u`** [SIMD arithmetic instruction](/en-US/docs/WebAssembly/Reference/SIMD/arithmetic) takes lanes 4–7 of two unsigned [`v128`](/en-US/docs/WebAssembly/Reference/Types/v128) `i16x8` value interpretations, multiplies the values in the corresponding lanes, and outputs the result of those operations into an `i32x4` value interpretation.
```

Author: OK. I've updated all instances to use this wording. I agree that it's a bit clearer.
